### DISCUSSION TITLE
Version 0.6.0 Beta 4

### DISCUSSION THREAD

**Discussion by DSheirer** [DSheirer/sdrtrunk#1711] [Announcements]
## **Application download links for each operating system are available below in the Assets section.**

**WARNING: Breaking Changes.**  This version changes the DMR decoder channel configuration's channel to frequency mapping format.  Previously, the user specified each Logical Slot Number (LSN) and the corresponding frequency and this required duplicate entries, one for each timeslot.  This has been updated to use the Logical Channel Number (LCN) and you only have to input one map entry for each repeater LCN.  The LCN and Frequency values can be found in the Radio Reference details for the DMR system. 

If you wish to continue using versions prior to 0.6.0 Beta 2, please make a backup copy of your playlist before you modify your current playlist to change the DMR channel to frequency maps.  That way you can switch between playlists.

## 1. Features:
* SDRPlay RSP tuner support  (API versions 3.07 - 3.11)
* Airspy Discovery HF+ support
* RTL2832 Support for R828D (aka rtl-sdr Blog V4) and FC0013 Tuners
* AM & NBFM Adaptive Squelch & AM Decoder Enhancements
* Digital Coded Squelch (DCS) aux decoder available with NBFM decoder
* LTR, LTR-Net & Passport Decoder enhancements
* rdio-scanner call broadcasting
* Update to OpenJDK 20 & Gradle 8.2

## 2. Requirements:
* Windows x86 (64-bit)
* Linux x86/aarch64 (64-bit)
* OSX x86/aarch64 (64-bit)

**Note**: JMBE library must be manually downloaded/built for OSX aarch64 due to OSX sandboxing of downloaded code.
**Note**: Linux for arm-32 is NOT currently supported

## 3. Optional Requirements
**JMBE version 1.0.9+** is required for decoding APCO25 Phase 1/2 and DMR digital audio on Linux and OSX aarch64 and this library can be compiled within the sdrtrunk application.

Legacy instructions for compiling the JMBE library are located at: **https://github.com/DSheirer/sdrtrunk/wiki/JMBE**

## 4. User Manual
* [Wiki](https://github.com/DSheirer/sdrtrunk/wiki)
* [User Manual](https://github.com/DSheirer/sdrtrunk/wiki/User-Manual)

## 5. Support
* [Support](https://github.com/DSheirer/sdrtrunk/wiki/Support)

## 6. What's Changed
* Update README.md by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1692
* #1694 SDRPlay RSP Memory Leak & Audio Playback Delays by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1695
* #1673 Broadcastify Call Streaming Config Editor - Can't Edit System ID by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1696
* #1665 Refresh Broadcastify Streams In Playlist Editor by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1697
* #1654 Playlist Channel Editor Frequency Column Sort by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1698
* #1653 Playlist Channel Edits With Applied Filter Produces Error by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1699
* #1651 Version 0.6.0 Beta 4 and Version Naming Conventions by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1700
* #1644 SDRPlay API Version 3.12 For Mac OSX by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1701
* #1704 Patch Group Formatting by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1705
* #1706 Github Action For Nightly Releases by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1707
* #1709 Nightly Release Build Timestamp Info by @DSheirer in https://github.com/DSheirer/sdrtrunk/pull/1710
* Explicitly send filename to Rdio-Scanner. by @tadscottsmith in https://github.com/DSheirer/sdrtrunk/pull/1708

**Full Changelog**: https://github.com/DSheirer/sdrtrunk/compare/v0.6.0-beta-3...v0.6.0-beta-4

<hr /><em>This discussion was created from the release <a href='https://github.com/DSheirer/sdrtrunk/releases/tag/v0.6.0-beta-4'>Version 0.6.0 Beta 4</a>.</em>

**Comment by galacticboy2009** [comment]
Thank you!
